test/helpers/setup-helper.js

```diff
@@ -89,10 +91,35 @@ window.document.body.appendChild(popoverContent);
 // eslint-disable-next-line no-shadow
 const { default: fetch, Headers, Request, Response } = require('node-fetch');
 
-Object.assign(window, { fetch, Headers, Request, Response });
+const shimmedFetch = (url, ...args) => {
```

Comment:
We need to shim fetch in order to support relative URLs fetch (which in a node environment translates to accessing filesystem files). This is used by the i18n provider to get the correct locale translation file (messages.json).